gruffalo-standalone.sh

```diff
@@ -2,4 +2,13 @@
 
 mvn dependency:build-classpath -Dmdep.outputFile=classpath -q
 CLASSPATH=target/classes:`cat classpath`
-java -cp $CLASSPATH -Xmx2024m -Xms2024m com.outbrain.gruffalo.StandaloneGruffaloServer "$@"
+# run com.outbrain.gruffalo.StandaloneGruffaloServer -h to check available args and usage
+# change port and cluster before starting
+java \
+  -cp $CLASSPATH \
+  -Djava.net.preferIPv4Stack=true \
+  -Xmx2024m \
+  -Xms2024m \
+  com.outbrain.gruffalo.StandaloneGruffaloServer \
+  --port 2003 \
+  --cluster node01:2003,node02:2003,node03:2003
```

Comment:
You removed the ability to pass in command line args, and replaced it with hard coded one